Cargo.toml

```diff
@@ -26,7 +26,8 @@ resolver = "3"
 dpdk = { path = "./dpdk" }
 dpdk-sys = { path = "./dpdk-sys" }
 dpdk-sysroot-helper = { path = "./dpdk-sysroot-helper" }
-errno = { path = "./errno" }
+dplane-rpc = { git = "https://github.com/githedgehog/dplane-rpc.git", branch = "master" }
```

Comment:
Why does this need to be in the top-level if only the routing crate will use it ?